src/playasophy/wonderdome/mode/strip_eq.clj

```diff
@@ -0,0 +1,75 @@
+(ns playasophy.wonderdome.mode.strip_eq
+  (:require
+    [playasophy.wonderdome.geometry.sphere :as sphere]
+    [playasophy.wonderdome.mode.core :as mode]
+    [playasophy.wonderdome.util.color :as color]
+  )
+)
+
+; StripEQ will process audio signals and turn each strip into an indicator for
+; the strength of a range of frequencies over time.
+(defrecord StripEqMode
+  [num_strips pix_per_strip]
```

Comment:
Another style pointer - Clojure uses kebab-case rather than snake_case for symbols and other names.